superset-frontend/src/SqlLab/components/QueryTable.jsx

```diff
@@ -100,43 +108,43 @@ const QueryTable = props => {
           </div>
         );
         q.user = (
-          <Button
+          <StyledButton
             buttonSize="small"
             buttonStyle="link"
             onClick={() => props.onUserClicked(q.userId)}
           >
             {q.user}
-          </Button>
+          </StyledButton>
         );
         q.db = (
-          <Button
+          <StyledButton
             buttonSize="small"
             buttonStyle="link"
             onClick={() => props.onDbClicked(q.dbId)}
           >
             {q.db}
-          </Button>
+          </StyledButton>
         );
         q.started = moment(q.startDttm).format('HH:mm:ss');
         q.querylink = (
-          <Button
+          <StyledButton
             buttonSize="small"
             buttonStyle="link"
             onClick={() => openQuery(q.queryId)}
           >
             <i className="fa fa-external-link m-r-3" />
             {t('Edit')}
-          </Button>
+          </StyledButton>
```

Comment:
it feels a little weird to make all of these buttons static. Can we put them in a wrapper that is static instead?

Reply:
I realized after pulling this down and looking at it that there isn't nothing necessarily significant with the `static` style, it just doesn't seem to like the current `relative` one. The changes that you made look good.